openshift/openshift/pom.xml

```diff
@@ -220,4 +220,19 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <systemPropertyVariables>
+            <sun.net.spi.nameservice.provider.1>dns,ArquillianCubeNameService</sun.net.spi.nameservice.provider.1>
```

Comment:
Sorry if I am missing something but where is `ArquillianCubeNameService` taken from?

Reply:
It comes from [ArqCubeNameServiceDescriptor implementation in openshift/openshift module](https://github.com/arquillian/arquillian-cube/blob/c7025efb2bda2f111af5864f2196255eff85d720/openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/dns/ArqCubeNameServiceDescriptor.java#L23).

Reply:
Ok, thanks @gaol. Now I am wondering now whether we should keep setting it in the [test class](https://github.com/arquillian/arquillian-cube/blob/c7025efb2bda2f111af5864f2196255eff85d720/openshift/openshift/src/test/java/org/arquillian/cube/openshift/impl/DnsServiceTest.java#L23). BTW - it is documented as well. Should we update and just keep passing the properties to the surefire plugin execution via `mvn`? WDYT?

Reply:
I think it is fine to leave it settings in the test classes, because it will make the test pass when you trying to run it in the IDE without having additional setup, but I am also good to delete the settings in the test class. :)

And the [NameService has been dropped from JDK9](https://bugs.openjdk.org/browse/JDK-8134577), so we may need to think about how to do with this when https://github.com/arquillian/arquillian-cube/issues/1270 needs to be fixed.

Reply:
+1, I'd say we can leave it as is now, i.e. accept your changes and resolve this thread.

It would be useful to track the future changes with an issue.